cmd/minishift/cmd/start.go

```diff
@@ -115,36 +108,16 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	proxyConfig := handleProxies()
 
-	machineConfig := cluster.MachineConfig{
-		MinikubeISO:      viper.GetString(startFlags.ISOUrl.Name),
-		Memory:           viper.GetInt(startFlags.Memory.Name),
-		CPUs:             viper.GetInt(startFlags.CPUs.Name),
-		DiskSize:         calculateDiskSizeInMB(viper.GetString(startFlags.DiskSize.Name)),
-		VMDriver:         viper.GetString(startFlags.VmDriver.Name),
-		DockerEnv:        dockerEnv,
-		InsecureRegistry: insecureRegistry,
-		RegistryMirror:   registryMirror,
-		HostOnlyCIDR:     viper.GetString(startFlags.HostOnlyCIDR.Name),
-		OpenShiftVersion: viper.GetString(startFlags.OpenshiftVersion.Name),
-		ShellProxyEnv:    shellProxyEnv,
-	}
-
-	fmt.Printf("Starting local OpenShift cluster using '%s' hypervisor...\n", machineConfig.VMDriver)
-
+	// to determine whether we need to run post cluster up actions,
```

Comment:
`to determine` to `Determine`. We have started comments with Caps (at least I am following it).